doc/source/reference/random/multithreading.rst

```diff
@@ -107,3 +103,11 @@ that does not use an existing array due to array creation overhead.
        ...: %timeit rg.standard_normal(10000000)
 
     Out[6]: 125 ms ± 309 µs per loop (mean ± std. dev. of 7 runs, 10 loops each)
+
+Note that if `threads` is not set by the user, it will be determined by the architecture
+
```

Comment:
I'm not sure this addition is necessary - the code defining `MultithreadedRNG` is pretty clear and a user should be able to grok how the number of threads is determined. Since the example doesn't highlight any of the `np.random` machinery itself I'd lean towards excluding this code-block.

Reply:
I would like to keep a reference in there. When I started working with multithreading and random generators, I stepped in the "common" trap of not thinking about what would happen to the random states. I think it is a common enough problem that (new) users are not aware of this, see eg. the discussion in #9650. What would you the others think @bashtage?

Reply:
Oops, I must have put the wrong comment in the wrong bubble - I agree this can stay, though what I meant to suggest was to replace "the architecture" with "multiprocessing.cpu_count".